### PR TITLE
Upgrade CCM to v2.7.0

### PR DESCRIPTION
**Pull request by shaoyongfeng**
**How to categorize this PR?**

/area control-plane
/kind enhancement
/platform alicloud

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes https://github.com/gardener/gardener-extension-provider-alicloud/issues/526

**Special notes for your reviewer**:

**Release note**:

```other operator
The following image is updated:
- registry.eu-central-1.aliyuncs.com/gardener-de/alibaba-cloud-controller-manager:v1.9.3-372 -> registry-eu-central-1.ack.aliyuncs.com/acs/cloud-controller-manager-amd64:v2.7.0
```

### CODE REVIEW

File: imagevector/images.yaml [hunk 1]
```diff
@@ -31,8 +31,8 @@ images:
       availability_requirement: 'low'
 - name: alicloud-controller-manager
   sourceRepository: https://github.com/kubernetes/cloud-provider-alibaba-cloud
-  repository: registry.eu-central-1.aliyuncs.com/gardener-de/alibaba-cloud-controller-manager
-  tag: v1.9.3-372 # the upstream image is using non-semver tag (registry.cn-shanghai.aliyuncs.com/acs/cloud-controller-manager-amd64:v1.9.3.372-gcf3535b-aliyun).
+  repository: registry-eu-central-1.ack.aliyuncs.com/acs/cloud-controller-manager-amd64
+  tag: v2.7.0
```
Comment (ialidzhikov) [inline imagevector/images.yaml:35]: There is [v2.8.0](https://github.com/kubernetes/cloud-provider-alibaba-cloud/releases/tag/v2.8.0) since 2 days.

**Review by ialidzhikov — COMMENTED** [review]
Could you please also add a release note?

File: imagevector/images.yaml (hunk 1 above)
Comment (shaoyongfeng) [reply imagevector/images.yaml:35]: Hi @ialidzhikov ,
the latest version is not widely adapted. for safety reason, we decide to use a stable version documented in https://help.aliyun.com/document_detail/94925.html?spm=a2c4g.11186623.6.576.66c95d626heOCg

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**